EIPS/eip-5069.md

```diff
@@ -0,0 +1,98 @@
+
+## Motivation
+The process for becoming an EIP editor is previously undocumented in the EIP repository itself.
+
+## Specification
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+### All PRs
+EIP editors MUST do the following for all new PRs submitted to the EIP repository:
+1. Review proposals: All updated proposals MUST pass a readiness check, unless this is unfeasable:
+    * The title describing the content MUST be accurate.
+    * The idea put forth MUST make sense technically.
+    * The content MUST not contain any language errors (spelling, grammar, and sentence structure).
+    * The proposal MUST use GitHub-flavored Markdown for formatting.
+    * The proposal MUST NOT have any blank sections, TODOs, or placeholders.
+    * The proposal MUST provide all required sections, and MUST NOT contain any sections not listed in [EIP-1](./eip-1.md).
+    * Any code provided MUST use best-practices for that programming language.
+    * The proposal MUST follow all latest [EIP-1](./eip-1.md) guidelines.
+1. Provide feedback: A review MUST be submitted, containing specific instructions to reject or improve the changes.
+1. Approve proposal: When the PR is ready to be merged, a review accepting the changes will result in the PR being merged.
+
+### New Proposals
+EIP editors MUST do the following for all new proposals submitted to the EIP repository:
+1. Ensure relevance: New proposals MUST be related to the Ethereum ecosystem.
+1. Assign EIP numbers: New proposals SHALL have an EIP number assigned, which SHOULD be the PR number. In some cases, to discourage squatting, an unused EIP number SHOULD be assigned instead.
+1. Perform the [All PRs](#all-prs) checklist
+
+### Other Changes
+Requirements for other PRs (status change, edits to a `final` EIP) are provided in [EIP-1](./eip-1.md).
+
+### Exceptional Cases / Miscellaneous Notes
+- When updating the status from `stagnant` to `draft`, it is up to the original author to bring it back to the status as they find appropriate. It MAY contain the same content as when its state was changed to `stagnant`, but MUST pass the readiness check. Note that the readiness check MAY have changed since the EIP was last updated - the EIP MUST be updated to use the latest standards!
+- When a proposal has been in the `stagnant` state for more than 6 months, then `stagnant` MAY be considered its terminal status. However, the original author MAY still update the EIP, changing the status from `stagnant` to `draft`.
+- If a proposal needs to be moved to `final` because the `last call` deadline was exceeded, but all authors are absent, an EIP champion MAY be added to the author list to deal with resulting comments. There are no requirements to become an EIP champion. Alternatively, the `last call` deadline may be extended.
```

Comment:
```suggestion
- If a proposal needs to be moved to `Final` because the `Last Call` deadline was exceeded, but all authors are absent, an EIP champion MAY be added to the author list to deal with resulting comments. There are no requirements to become an EIP champion. Alternatively, the `last call` deadline may be extended.
```